<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="c_title">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>10</width>
      <height>100</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;腾讯内部签入签出系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTimeEdit" name="startTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="endTime">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>签入时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>签出时间：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>周一</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>周二</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>周四</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>周三</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>周六</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>周五</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="day7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>周日</string>
    </property>
   </widget>
   <widget class="QToolButton" name="startTask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(18, 186, 0)</string>
    </property>
    <property name="text">
     <string>定时任务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkIn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>签入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkOut">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>签出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>startTask</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>on_startTask_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkIn</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>on_checkIn_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkOut</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>on_checkOut_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startTime</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>Main</receiver>
   <slot>on_startTime_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endTime</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>Main</receiver>
   <slot>on_endTime_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>day1</sender>
   <signal>clicked(bool)</signal>
   <receiver>Main</receiver>
   <slot>on_day1_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>day2</sender>
   <signal>clicked(bool)</signal>
   <receiver>Main</receiver>
   <slot>on_day2_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>day3</sender>
   <signal>clicked(bool)</signal>
   <receiver>Main</receiver>
   <slot>on_day3_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>day4</sender>
   <signal>clicked(bool)</signal>
   <receiver>Main</receiver>
   <slot>on_day4_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>day5</sender>
   <signal>clicked(bool)</signal>
   <receiver>Main</receiver>
   <slot>on_day5_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>day6</sender>
   <signal>clicked(bool)</signal>
   <receiver>Main</receiver>
   <slot>on_day6_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>day7</sender>
   <signal>clicked(bool)</signal>
   <receiver>Main</receiver>
   <slot>on_day7_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_startTask_click()</slot>
  <slot>on_checkIn_click()</slot>
  <slot>on_checkOut_click()</slot>
  <slot>on_startTime_click()</slot>
  <slot>on_endTime_click()</slot>
  <slot>on_day1_click()</slot>
  <slot>on_day2_click()</slot>
  <slot>on_day3_click()</slot>
  <slot>on_day4_click()</slot>
  <slot>on_day5_click()</slot>
  <slot>on_day6_click()</slot>
  <slot>on_day7_click()</slot>
 </slots>
</ui>
